firmware/quadruna/VC/src/app/states/app_allStates.c

```diff
@@ -60,13 +60,16 @@ void app_allStates_runOnTick100Hz(void)
     // app_sbgEllipse_broadcast();
 
     // Set status to false (which blocks drive) if either inverter is faulted, or another board has set a fault.
+    return true;
 }
 
-void app_allStates_runOnTick1Hz(void)
+bool app_allStates_runOnTick1Hz(void)
```

Comment:
This just a code thing but if we are always gonna return true why not just leave it as void. Ik BMS does this and I am not sure why